Assets/MixedRealityToolkit.Services/InputSystem/FocusProvider.cs

```diff
@@ -39,7 +39,7 @@ float IMixedRealityFocusProvider.GlobalPointingExtent
         {
             get
             {
-                if (MixedRealityToolkit.HasActiveProfile &&
+                if (MixedRealityToolkit.Instance.HasActiveProfile &&
```

Comment:
Can instance be null?

Reply:
If memory serves, if Instance is null and the component is in the scene, it will be instantiated. The core pattern has not changed (see other examples of calling through .Instance).

Thanks!